pkgs/by-name/tr/tracexec/package.nix

```diff
@@ -15,26 +15,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tracexec";
-  version = "0.12.0";
+  version = "0.13.0-unstable-2025-09-07";
 
   src = fetchFromGitHub {
     owner = "kxxt";
     repo = "tracexec";
-    tag = "v${version}";
-    hash = "sha256-j1zgHDO5bmJAXi9KvkHqenm/QfM9DmD9yNqF6TxJ9sY=";
+    rev = "dbb9b733370f5200df2a0de7f007312c23431480";
+    hash = "sha256-M2ZIfWupnFxQZvr5cl8V0xtLgh+xBcaHHVsHIoio7nI=";
   };
 
-  # remove if updating to rust 1.85
-  postPatch = ''
-    substituteInPlace Cargo.toml \
-      --replace-fail "[package]" ''$'cargo-features = ["edition2024"]\n[package]' \
-      --replace-fail 'rust-version = "1.85"' ""
-  '';
-
   # remove if updating to rust 1.85
```

Comment:
I know it's unrelated, so this comment is not blocking, but if you could remove this one too that'd be great 😄